live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -29,6 +29,12 @@
     msg: "Required environment variable 'APPLIANCE_USERNAME' is empty."
   when: lookup('env', 'APPLIANCE_USERNAME') == ''
 
+#
+# Apt cache needs to be up to date.
+#
+- apt:
+    update_cache: yes
```

Comment:
Could you explain why this change is necessary? What problem does this solve?

Reply:
An apt task would fail next during my appliance-build orchestrator run. It would fail to find a dependency on some repository (don't recall which one it was exactly).